web/src/lib/utils/analytics-helpers.ts

```diff
@@ -64,16 +64,11 @@ export const phaseChangeAnalytics = ({
     return;
   } else if (
     oldProfileData.operatingPhase === "NEEDS_TO_FORM" &&
-    newProfileData.operatingPhase === "NEEDS_TO_REGISTER_FOR_TAXES"
+    newProfileData.operatingPhase === "FORMED"
```

Comment:
How are we addressing the jump from needs business structure to formed, I'd say they are a GP business structure